pkg/scheduler/api/job_info.go

```diff
@@ -652,6 +652,36 @@ func (ji *JobInfo) CheckTaskMinAvailable() bool {
 	return true
 }
 
+// CheckTaskMinAvailableReady return ready pods meet task minavaliable.
+func (ji *JobInfo) CheckTaskMinAvailableReady() bool {
+	occupiedMap := map[TaskID]int32{}
+	for status, tasks := range ji.TaskStatusIndex {
+		if AllocatedStatus(status) ||
+			status == Succeeded ||
+			status == Pipelined {
+			for _, task := range tasks {
+				occupiedMap[getTaskID(task.Pod)] += 1
+			}
+			continue
+		}
+
+		if status == Pending {
```

Comment:
Should `status == Pending` check remove in the  `jobReadyFn` function?

Reply:
l think it should keep the original in this pr